docker-compose.yml

```diff
@@ -156,6 +156,100 @@ services:
       cp /root/.lnd/macaroons.db /root/.lnd/data/chain/bitcoin/regtest/macaroons.db
       cp /root/.lnd/admin.macaroon /root/.lnd/data/chain/bitcoin/regtest/admin.macaroon
       /bin/lnd
+  loopserver:
+    image: lightninglabs/loopserver:v0.9.52-beta
+    restart: unless-stopped
+    volumes:
+      - ${PWD}/dev/lnd/lnd.conf:/root/.lnd/lnd.conf
+      - ${PWD}/dev/lnd/tls.key:/root/.lnd/tls.key
+      - ${PWD}/dev/lnd/tls.cert:/root/.lnd/tls.cert
+      - ${PWD}/dev/lnd/regtest/lnd-outside-1.admin.macaroon:/root/.lnd/data/chain/bitcoin/regtest/admin.macaroon
```

Comment:
can't you bind the directory instead?

Reply:
yeah i will have to modify openoms script `dev/bin/save-lnd-data.sh` . right now the script does not export all the required macaroons for loopserver (readonly, invoices, signer, chainnotifier, router and walletkit).